diagnostic_feedback/tests/integration/base_test.py

```diff
@@ -0,0 +1,183 @@
+from selenium.webdriver.support.ui import WebDriverWait
+from xblock.fields import String
+from xblockutils.base_test import SeleniumXBlockTest
+from xblockutils.resources import ResourceLoader
+import time
```

Comment:
@jareerahsan Please reorder these imports to follow the [imports order](https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order) specified in edX's Python guidelines.